Mono.Addins.MSBuild/Mono.Addins.MSBuild.csproj

```diff
@@ -34,28 +30,16 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="System" />
-    <Reference Include="Microsoft.Build.Framework" />
-    <Reference Include="Microsoft.Build.Utilities.v4.0" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="AssemblyInfo.cs" />
-    <Compile Include="ResolveAddinReferences.cs" />
-    <Compile Include="Util.cs" />
-  </ItemGroup>
-  <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.targets" />
-  <ItemGroup>
-    <ProjectReference Include="..\Mono.Addins.Setup\Mono.Addins.Setup.csproj">
-      <Project>{A85C9721-C054-4BD8-A1F3-0227615F0A36}</Project>
-      <Name>Mono.Addins.Setup</Name>
-      <Private>False</Private>
-    </ProjectReference>
     <ProjectReference Include="..\Mono.Addins\Mono.Addins.csproj">
       <Project>{91DD5A2D-9FE3-4C3C-9253-876141874DAD}</Project>
       <Name>Mono.Addins</Name>
       <Private>False</Private>
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <None Include="Mono.Addins.targets" />
+    <Compile Remove="ResolveAddinReferences.cs" />
```

Comment:
Why is the file being removed from compilation?

Reply:
Change Mono.Addins.Setup to netstandard2.0 aswell and included ResolveAddinReferences.cs again.